srcpkgs/pyradio/template

```diff
@@ -1,17 +1,21 @@
 # Template file for 'pyradio'
 pkgname=pyradio
-version=0.8.7.3
+version=0.8.9.9
 revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3"
+depends="python3 python3-requests python3-dnspython"
```

Comment:
you don't need to list python3 as a dependency when the package already depends on other python packages (because they depend on python3)

Reply:
Done!